third-party/sysdeps/common/sqlite3/CMakeLists.txt

```diff
@@ -40,8 +40,9 @@ endif()
 
 # Otherwise, this is the sub-project build.
 cmake_minimum_required(VERSION 3.25)
-project(SQLITE3_BUILD)
+project(SQLITE3_BUILD VERSION 3.49.1)
```

Comment:
I think this needs to be raised as we recently bumped sqlite3 https://github.com/ROCm/TheRock/commit/a4d180476c9518eb22c36076079d25e26f943dde.